Gemfile

```diff
@@ -21,6 +21,7 @@ gem 'jbuilder', '~> 2.7'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
+gem 'haml-rails', '~> 2.0'
```

Comment:
I like to add a little comment stating the purpose of each gem before the gem itself.